src/graphql/public/root/query/is-flash-npub-query.ts

```diff
@@ -0,0 +1,32 @@
+import { GT } from "@graphql/index"
+import { Accounts } from "@app"
+import {
+  IsFlashNpubInput,
+  IsFlashNpubPayload,
+} from "@graphql/public/types/object/is-flash-npub"
+import Npub from "@graphql/shared/types/scalar/npub"
+
+const IsFlashNpubQuery = GT.Field({
+  type: IsFlashNpubPayload,
+  args: {
+    input: {
+      type: GT.NonNull(IsFlashNpubInput),
+    },
+  },
+  resolve: async (_, args) => {
+    const {
+      input: { npub },
+    } = args
+    if (npub instanceof Error) {
+      throw npub
+    }
+    const output = await Accounts.findByNpub(npub)
+    console.log("output is", output)
+    if (output instanceof Error) return { isFlashNpub: false }
```

Comment:
Seems a mistake to return a 200 with a `false` response when an error actually occurs

Reply:
ahh yes, my hope was i'm catching accountnotfounderrors, but this is generic, i'll add it to specifically capture those errors and return a false only then, thanks!